Assets/HoloToolkit/UX/Resources/Models/HMD/hololens_low.fbm/PlasticMaterial.fx

```diff
@@ -0,0 +1,7 @@
+//---------------------------------------------------------------------------
```

Comment:
What's up with this .fx asset?

Reply:
I'm not sure! It seems like it's getting auto-generated, but I don't know how or why. @keveleigh noticed this in the other pull request as well. Theories, anyone?

Reply:
Disable the material import on the mesh asset.

Reply:
That's where we're stumped, it's already disabled.

Reply:
Could we remove this, and it's meta file until we get it sorted out?

Reply:
Yup, removed.